packages/images-plugin/lib/plugin.js

```diff
@@ -83,6 +82,36 @@ export default class ImageElementPlugin extends BasePlugin {
     // otherwise just do the normal thing
     return await next();
   }
+
+  async performActions(next, driver, ...args) {
+    // Replace with coordinates when ActionSequence includes image elements.
+    const [actionSequences] = /** @type {[import('@appium/types').ActionSequence[]]} */ (args);
+    for (const actionSequence of actionSequences) {
+      for (const action of actionSequence.actions) {
+        // The actions that can have an Element as the origin are "pointerMove" and "scroll".
+        if (
+          (action.type === 'pointerMove' || action.type === 'scroll') &&
+          !_.isNil(action.origin) &&
+          !_.isString(action.origin)
+        ) {
+          const elId = util.unwrapElement(action.origin);
+          if (_.isString(elId) && elId.startsWith(IMAGE_ELEMENT_PREFIX)) {
```

Comment:
this might be reduced to ```
if (!_.startsWith(elId, IMAGE_ELEMENT_PREFIX)) { 
  continue; 
}
...
```

Reply:
same comments could be applied to the above condition. In general, it is easier to scan through guards rather than log nested blocks, for example:

```
for (const item in items) {
   if (!condition(item)) {
      continue;
   }
   // do something with item
}
```

over

```
for (const item in items) {
   if (condition(item)) {
      // do something with item
   }
}
```